### ISSUE TITLE
Feature Request: Support for changing Treat Wounds macro DCs

### ISSUE THREAD

**Issue by thes33** [foundryvtt/pf2e#6813]
It seems like the current DCs are hard-coded.  It would be nice to expose these values in some way to allow changing the DCs. (Especially for those using Proficiency Without Level).

**Comment by thes33** [comment]
I'm not sure this would expressly be considered a "bug", since its for a variant system that isn't fully fleshed out by Paizo.  I will say that the PWL simple DCs are quite punitive for using Treat Wounds, which is why I would request the ability to change these values directly.  I've been using a custom set of Treat Wounds DCs and would like to be able to still use the convenient Treat Wounds Macro, but I can't seem to find where these values are stored.


<table>
<thead>
  <tr>
    <th>Simple DCs</th>
    <th>Normal DC</th>
    <th>PWL DC</th>
    <th>Custom PWL DC</th>
    <th>Custom Treat Wounds DC</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td>Untrained</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td></td>
  </tr>
  <tr>
    <td>Trained</td>
    <td>15</td>
    <td>15</td>
    <td>15</td>
    <td>12</td>
  </tr>
  <tr>
    <td>Expert</td>
    <td>20</td>
    <td>20</td>
    <td>18</td>
    <td>16</td>
  </tr>
  <tr>
    <td>Master</td>
    <td>30</td>
    <td>25</td>
    <td>22</td>
    <td>19</td>
  </tr>
  <tr>
    <td>Legendary</td>
    <td>40</td>
    <td>30</td>
    <td>25</td>
    <td>22</td>
  </tr>
</tbody>
</table>

**Comment by TikaelSol** [comment]
Since this would require us to guess what the PWL DC is we won't implement pre-set ones, but maybe a way to customize them more easily is possible. The entire macro needs basically rebuilt to work with RE support anyways.

**Comment by thes33** [comment]
Yes, customization of the DC's would be great.  The macros work so well and we miss using them.

**Comment by 7H3LaughingMan** [comment]
> Since this would require us to guess what the PWL DC is we won't implement pre-set ones, but maybe a way to customize them more easily is possible. The entire macro needs basically rebuilt to work with RE support anyways.

I wanted to follow up on this, doesn't the system already have something in place for calculating simple DCs? You just have to do a check to determine if PWOL is enabled and it can provide the appropriate DC.

https://github.com/foundryvtt/pf2e/blob/v13-dev/src/module/dc.ts
